packages/compass-connections/src/stores/connections-store.tsx

```diff
@@ -332,13 +332,7 @@ export function useConnections({
         // noop, we're already logging in the connect method
       });
     }
-
-    return () => {
```

Comment:
Forgot to clean this up in the previous PR, compass-connections top level provider now closes all connections on unmount